src/Controller/JsonController.php

```diff
@@ -65,20 +66,24 @@ public function checkIn(Guest $guest): JsonResponse
         ]);
     }
     /**
-     * @Route("/json/checkout/{guest}", name="app_json_checkout")
+     * @Route("/json/checkout/{guest}", name="app_json_checkout", methods={"POST"})
      */
     public function checkOut(Guest $guest): JsonResponse
     {
-        $status = 'NACK';
         try {
             $guest->setCheckInTime(null);
+            $guest->setCheckedInPluses(0);
```

Comment:
hier wäre NULL besser. In den bisherigen Templates prüfen wir öfter mal if(varName), keine AHnung, wie sich das mit 0 verhält

Reply:
Done